
<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <!-- <link rel="icon" href="../../../../favicon.ico"> -->

    <title>BART help</title>
    <link rel="shortcut icon" href="{{ url_for('static', filename='bart_icon.png') }}">

    <!-- Bootstrap core CSS -->
    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" rel="stylesheet">
    <!-- Custom styles for this template -->
    <link href="{{ url_for('static', filename='sticky-footer-navbar.css') }}" rel="stylesheet">

    <!-- change all the fonts inside class='container' -->
    <style>
        .container {
          font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
        }
    </style>

  </head>

  <body class="bg-light">
  {% include "header.html" %}
    <div class="container">
        <div class="py-3">
          <h3><font color="#002F6C">BARTweb help page</font></h3>
          <p>BARTweb is an interactive web server to predict functional transcriptional regulators that regulate a given gene list or associate with a genomic profile. It is a web interface to <a href="https://github.com/zanglab/bart2" target="_blank"> BART Python package</a>.</p>
          <p>If you use BARTweb in your data analysis or publish the results of BARTweb, please cite the following paper in the main text of your manuscript: </p>
          <ul>
          <li>Zhenjia Wang, Mete Civelek, Clint Miller, Nathan Sheffield, Michael J. Guertin, Chongzhi Zang.  BART: a transcription factor prediction tool with query gene sets or epigenomic profiles. <em>Bioinformatics</em>, <strong>34</strong>, 2867–2869 (2018). DOI: <a href='https://doi.org/10.1093/bioinformatics/bty194' target="_new">10.1093/bioinformatics/bty194</a></li>

          </ul>

          <!-- Add tutorial video link -->
          <h3>Tutorial Video</h3>
          <iframe width="720" height="405" src="https://www.youtube.com/embed/ChBELOHfWC4"></iframe>

         
          <h3>Manual</h3>
          <p><img src="{{ url_for('static', filename='bartweb_guidelines_bart2.png') }}" alt='bartweb_guidelines' referrerPolicy='no-referrer' width="1000"/></p>
          <h5>Submit a job</h5>
          <p>The first three options (species, data type and input data) are required. </p>
          <p><strong>Species</strong>: Currently, only human (hg38) and mouse (mm10) genomes are supported. </p>
          <p><strong>Input data type</strong>: 
            <ul>
            <li><strong>Gene list</strong>: A gene list has to be official gene symbols (HGNC for human or MGI for mouse) in text format. BARTweb can identify functional Transcription Regulators (TRs) that regulate the gene set. At least 100 genes are recommended in the input.
            </ul>

            <ul>
            <li><strong>ChIP-seq data</strong>: ChIP-seq data can be mapped reads in either BAM or BED format. BARTweb can identify TRs have a binding profile correlated with a ChIP-seq profile. At least 1 million reads are recommended in the input.
            </ul>

            <ul>
            <li><strong>Region (under BETA testing)</strong>: The region input has to be a BED file with scored, non-overlapping intervals, for example, ChIP-seq peaks. The scores have to be on 5<sup>th</sup> row of the BED file. Please make sure that the ChIP-seq data or scored bed file are mapped to the correct genome version (hg38 or mm10). BARTweb can identify TRs enriched in a genomic region set.
            </ul>


          <p>If an email address is provided, BARTweb will send email notifications with the job key, job status, and the result URL.</p>
          <p>After clicking the “Submit” button, BARTweb will redirect to the result page.</p>
          <br>
          <h5>Result page</h5>
          <p>It usually takes a few minutes to run a job. You can click the “Result” button on the navigation bar or use the job key to get the result or the job status anytime. If the job has not finished, you will see the “still-running” status in the title and a processing log presented. You can click the “Copy Key” button to copy the job key to the clipboard and keep it for your record. If the job fails, the detailed error message will be shown in the processing log. If you cannot figure out possible causes of the error, you are welcome to contact us with the job key for help.</p>
          <p>You can share your BART analysis result with others by providing them with the result link URL or the job key. Anyone with a result URL or a job key can open and view the BART analysis result at the user’s own risk.</p>
          <p>For successfully finished jobs, the analysis result will be stored on the web server and can be retrieved with the job key for 180 days after the job submission.</p>
          <br>
          <h4>How to interpret the BART result</h4>
          <p><img src="{{ url_for('static', filename='bartweb2_result.png') }}" alt='bartweb_result' referrerPolicy='no-referrer' width="1000"/></p>
          <br>
          <h5>Output 1: Transcriptional regulator (TR) prediction table</h5>
          <p>We take this sub-table of a <a href="{{ url_for('sample_result', sample_type='genelist')}}">pre-run result</a> (from a gene set that were downregulated upon OCT4(POU5F1) knocked down in a human embryonic stem cell line as an example to illustrate. Another <a href="{{ url_for('sample_result', sample_type='ChIPdata')}}">pre-run result</a> (from a ChIP-seq profile upon AR being enhanced in a human prostatic carcinoma cell line (LNCaP)) is also provided as reference for ChIP-seq data input.</p>
          <ul>
          <li><strong>TR</strong> (Column 1): Transcriptional regulator name. When clicking a TR name, the analysis plots for this TR will be shown in a pop-up window.</li>
          </ul>
          <ul>
          <li><strong>Wilcoxon test statistic &amp; P-value</strong> (Columns 2 &amp; 3): These two values indicate the level of association of each TR under the background of all other TRs. For each TR, we use Wilcoxon rank-sum test to compare the association scores from all ChIP-seq datasets for that regulator with the association scores from all ChIP-seq datasets for other TRs.</li>
          </ul>
          <ul>
          <li><strong>Z-score</strong> (Column 4): This value is to assess the specificity of each TR compared with a background model. We build up background models using the Wilcoxon test statistics obtained from all annotated gene sets from the Molecular Signatures Database (MSigDB) for the Genelist mode or all H3K27ac ChIP-seq datasets from the data compendium for ChIP-seq data mode, respectively.</li>
          </ul>
          <ul>
          <li><strong>Max AUC</strong> (Column 5): The maximum association score among multiple ChIP-seq datasets of that TR.</li>
          </ul>
          <ul>
          <li><strong>Relative Rank</strong> (Column 6): The average rank of Wilcoxon test statistic, Z-score and maxAUC for each TR, divided by the total number of TRs.</li>
          </ul>
          <ul>
          <li><strong>Irwin-Hall P-value</strong> (Column 7): This P-value indicates the integrative rank significance, using the Irwin-Hall distribution as the null distribution for unrelated ranks. One can use 0.01 as a threshold for significant TR identification.</li>
          </ul>
          <br>
          <h5>Output 2: Data files and figure images</h5>
          <p>Besides the transcriptional regulator table, other output data files are also available for download, including the predicted genomic regulatory (cis-regulatory element) profile and analysis plots for each regulator. The image files are in high resolution with publication quality.</p>
          <ul>
          <li><strong>*_adaptive_lasso_Info.txt</strong> provides regression information tells which representative H3K27ac samples are selected along with coefficients through adaptive lasso regression and sample annotations including cell line, cell type or tissue type. This is the output only generated in geneset mode.</li>
          </ul>
          <ul>
          <li><strong>*_CRE_prediction_lasso.txt</strong> is the predicted cis-regulatory profile of the input gene set and is a ranked list of all CREs (UDHS) in the genome. The higher the score, the more likely the regulatory element regulates the input gene set. This is the output only generated in geneset mode.</li>
          </ul>
          <ul>
          <li><strong>*_auc.txt</strong> provides the association score of each of the TR ChIP-seq dataset with the genome cis-regulatory profile.</li>
          </ul>
                      
        </div>

        {% include "footer.html" %}
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
    <script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery-slim.min.js"><\/script>')</script>
    <!-- <script src="../../assets/js/vendor/popper.min.js"></script> -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
    <!-- <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/js/bootstrap.min.js"></script> -->
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/js/bootstrap.min.js" integrity="sha384-o+RDsa0aLu++PJvFqy8fFScvbHFLtbvScb8AjopnFD+iEQ7wo/CG0xlczd+2O/em" crossorigin="anonymous"></script>
    <script src="{{ url_for('static', filename='holder.min.js') }}"></script>
    
  </body>
</html>
