
<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">
    <!-- <link rel="icon" href="../../../../favicon.ico"> -->

    <title>BART help</title>
    <link rel="shortcut icon" href="{{ url_for('static', filename='bart_icon.png') }}">

    <!-- Bootstrap core CSS -->
    <link href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" rel="stylesheet">
    <!-- Custom styles for this template -->
    <link href="{{ url_for('static', filename='sticky-footer-navbar.css') }}" rel="stylesheet">

    <!-- change all the fonts inside class='container' -->
    <style>
        .container {
          font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
        }
    </style>

  </head>

  <body class="bg-light">
  {% include "header.html" %}
    <div class="container">
        <div class="py-3">
          <h3><font color="#002F6C">BARTweb help page</font></h3>
          <p>BARTweb is an interactive web server to predict functional transcription factors that regulate a given gene list or associate with a genomic profile. It is a web interface to <a href='http://faculty.virginia.edu/zanglab/bart/index.htm'>BART Python package</a>. </p>
          <p>If you use BARTweb in your data analysis or publish the results of BARTweb, please cite the following paper in the main text of your manuscript: </p>
          <ul>
          <li>Zhenjia Wang, Mete Civelek, Clint Miller, Nathan Sheffield, Michael J. Guertin, Chongzhi Zang.  BART: a transcription factor prediction tool with query gene sets or epigenomic profiles. <em>Bioinformatics</em>, <strong>34</strong>, 2867–2869 (2018). DOI: <a href='https://doi.org/10.1093/bioinformatics/bty194' target="_new">10.1093/bioinformatics/bty194</a></li>

          </ul>
         
          <h3>Manual</h3>
          <p><img src="{{ url_for('static', filename='bartweb_guidelines_bart2.png') }}" alt='bartweb_guidelines' referrerPolicy='no-referrer' width="1000"/></p>
          <h5>Submit a job</h5>
          <p>The first three options (species, data type and input data) are required. </p>
          <p><strong>Species</strong>: Currently, only human (hg38) and mouse (mm10) genomes are supported. </p>
          <p><strong>Input data type</strong>: A gene list has to be official gene symbols in text format. ChIP-seq data can be mapped reads in either BAM or BED format. Please make sure that the ChIP-seq data are mapped to the correct genome version (hg38 or mm10).</p>
          <p>If an email address is provided, BARTweb will send email notifications with the job key, job status, and the result URL.</p>
          <p>After clicking the “Submit” button, BARTweb will redirect to the result page.</p>
          <br>
          <h5>Result page</h5>
          <p>It usually takes a few minutes to run a job. You can click the “Result” button on the navigation bar or use the job key to get the result or the job status anytime. If the job has not finished, you will see the “still-running” status in the title and a processing log presented. You can click the “Copy Key” button to copy the job key to the clipboard and keep it for your record. If the job fails, the detailed error message will be shown in the processing log. If you cannot figure out possible causes of the error, you are welcome to contact us with the job key for help.</p>
          <p>You can share your BART analysis result with others by providing them with the result link URL or the job key. Anyone with a result URL or a job key can open and view the BART analysis result at the user’s own risk.</p>
          <p>For successfully finished jobs, the analysis result will be stored on the web server and can be retrieved with the job key for 180 days after the job submission.</p>
          <br>
          <h4>How to interpret the BART result</h4>
          <p><img src="{{ url_for('static', filename='bartweb2_result.png') }}" alt='bartweb_result' referrerPolicy='no-referrer' width="1000"/></p>
          <br>
          <h5>Output 1: Transcription factor prediction table</h5>
          <p>We take this sub-table of a <a href='http://bartweb.org/sample_result/genelist'>pre-run result</a> as an example to illustrate.</p>
          <ul>
          <li><strong>TF</strong> (Column 1): Transcription factor name. When clicking a TF name, the analysis plots for this TF will be shown in a pop-up window.</li>
          </ul>
          <ul>
          <li><strong>Wilcoxon test statistic &amp; P-value</strong> (Columns 2 &amp; 3): These two values indicate the level of association of each TF under the background of all other TFs. For each TF, we use Wilcoxon rank-sum test to compare the association scores from all ChIP-seq datasets for that factor with the association scores from all ChIP-seq datasets for other TFs.</li>
          </ul>
          <ul>
          <li><strong>Z-score</strong> (Column 4): This value is to assess the specificity of each TF compared with a background model. We build up background models using the Wilcoxon test statistics obtained from all annotated gene sets from the Molecular Signatures Database (MSigDB) for the Genelist mode or all H3K27ac ChIP-seq datasets from the data compendium for ChIP-seq data mode, respectively.</li>
          </ul>
          <ul>
          <li><strong>Max AUC</strong> (Column 5): The maximum association score among multiple ChIP-seq datasets of that TF.</li>
          </ul>
          <ul>
          <li><strong>Relative Rank</strong> (Column 6): The average rank of Wilcoxon test statistic, Z-score and maxAUC for each TF, divided by the total number of TFs.</li>
          </ul>
          <ul>
          <li><strong>Irwin-Hall P-value</strong> (Column 7): This P-value indicates the integrative rank significance, using the Irwin-Hall distribution as the null distribution for unrelated ranks. One can use 0.01 as a threshold for significant TF identification.</li>
          </ul>
          <br>
          <h5>Output 2: Data files and figure images</h5>
          <p>Besides the transcription factor table, other output data files are also available for download, including the predicted genomic regulatory (enhancer) profile and analysis plots for each factor. The image files are in high resolution with publication quality.</p>
                      
        </div>

        {% include "footer.html" %}
    </div>

    <!-- Bootstrap core JavaScript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
    <script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery-slim.min.js"><\/script>')</script>
    <!-- <script src="../../assets/js/vendor/popper.min.js"></script> -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
    <!-- <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/js/bootstrap.min.js"></script> -->
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/js/bootstrap.min.js" integrity="sha384-o+RDsa0aLu++PJvFqy8fFScvbHFLtbvScb8AjopnFD+iEQ7wo/CG0xlczd+2O/em" crossorigin="anonymous"></script>
    <script src="{{ url_for('static', filename='holder.min.js') }}"></script>
    
  </body>
</html>
